app/src/main/res/layout/activity_main.xml

```diff
@@ -21,12 +22,14 @@
     <fragment
         android:id="@+id/nav_host_fragment"
         android:name="androidx.navigation.fragment.NavHostFragment"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_width="406dp"
+        android:layout_height="635dp"

```

Comment:
remove these fixed values

Reply:
Actually, I don't think you can remove this. Change it to 0dp to indicate that the size is being changed somewhere else.

Reply:
Even better use "match_parent" again

Reply:
match_parent for the height leaves a lot of white space at the top.

Reply:
Ahh, I see. I believe this is caused by `android:paddingTop="?attr/actionBarSize"` on line 7. Try removing that line and keep "match_parent" for height.